ImperatorToCK3/Source/Imperator/Countries/CountryFactory.cpp

```diff
@@ -0,0 +1,83 @@
+#include "Log.h"
+#include "ParserHelpers.h"
+#include "CommonRegexes.h"
+#include "CountryFactory.h"
+#include "CountryCurrencies.h"
+#include "CountryName.h"
+
+
+Imperator::Country::Factory::Factory()
+{
+	registerSetter("tag", country->tag);
```

Comment:
Setters don't work with factories, unfortunately. The captured variable gets set at the time the constructor runs, when the country is an invalid pointer, and it doesn't get updated when you create the country during the getCountry() call. This caused a segfault. I don't know why that's different than having the lambda be there directly, but it is.

Reply:
That's unfortunate. Your explanation makes sense. The difference is probably that in the registerSetter the lamdba captures the variable, while in registerKeyword the lambda usually captures `this`.

Reply:
Changed to registerKeyword ✔️